plugins/services/src/js/components/ServiceBreadcrumbs.js

```diff
@@ -119,7 +120,9 @@ class ServiceBreadcrumbs extends React.Component {
       return null;
     }
 
-    const serviceStatus = service.getStatus();
+    const serviceStatus = isFrameworkDeleting(service)
```

Comment:
How do you think if we add `isDeleeting()` method to the `Service.js` struct so we never need to check if it's a framework or not?

Then we could just say `service.isDeleting()` and having everything else that is not a `Framework` returns `false`;

Reply:
@orlandohohmeier ^^

Reply:
@nLight @orlandohohmeier I thought about that, but I then thought we would be returning information that is not true. If someone called service.isDeleting() on a non-framework service it could return wrong information because it could be actually deleting but we're returning false.
So instead of creating an unreliable method and since this task is for frameworks only I chose to make it framework specific.

Reply:
I'm wondering why we're not adding a new ` isDeleting` getter instead of adding a new service statuts as mentioned in https://github.com/dcos/dcos-ui/pull/2440#discussion_r143800283

We could overwrite `getServiceStatus` to return _deleting_ in case a framework is flagged with the respective env. Using the approach would allow us to eventually add the same feature for all services as we don't add a special framework case. It's also not presenting wrong data as we're only returning _deleting_ for frameworks. Furthermore I'd say it's a much cleaner API as we present only one method to get the service status.

Reply:
@nLight @orlandohohmeier I went with the isDeleting getter inside of the Service that we can extend in the future. My first approach was to make this very explicit but this is ok as well.

Reply:
Hey @orlandohohmeier, we're ready to merge this one, feel free to press the button if the solution looks good to you.

Reply:
@bstavroulakis @nLight I hesitate to merge the PR, as I still feel that we should use `getServiceStatus` instead of adding a new `isDeleting` getter. Though I'm ok to move forward if one of you can answer my question and tell me why need the new getter and what the difference between "deleting" and "deploying" is.

Reply:
@nLight @orlandohohmeier I removed the getter and I'm using only the getServiceStatus() now. You can give it a new swirl.